homeassistant/components/automation/state.py

```diff
@@ -24,6 +24,31 @@
 CONF_FOR = "for"
 
 
+def get_time_config(config):
+    """ Helper function to extract the time specified in the config """
+    if CONF_FOR in config:
```

Comment:
Can you rewrite this to be a guard clause too?

```python
if CONF_FOR not in config:
    return None
```

Also, can you return `None` instead of `False`.